package.json

```diff
@@ -21,7 +21,9 @@
     "code": "1.x.x"
   },
   "scripts": {
-    "test": "make test-cov"
+    "test": "node_modules/lab/bin/lab -a code -L",
+    "test-cov": "node_modules/lab/bin/lab -a code -t 100 -L",
+    "test-cov-html": "node_modules/lab/bin/lab -a code -r html -o coverage.html -L"
```

Comment:
it should be unnecessary to add `node_modules/lab/bin/` here – `node_modules/.bin/` is automatically on path.